### PR TITLE
Add mainQueue

### PR DESCRIPTION
**Pull request by zenangst**

### CODE REVIEW

File: Source/Functions.swift [hunk 1]
```diff
@@ -12,3 +12,9 @@ public func delay(delay:Double, closure: () -> Void) {
 public func localizedString(key: String, comment: String? = nil) -> String {
   return NSLocalizedString(key, comment: (comment != nil) ? comment! : key)
 }
+
+public func mainQueue(closure: () -> Void) {
```
Comment (RamonGilabert) [inline Source/Functions.swift]: Maybe dispatchInMainQueue would be a better name? :pig:

**Comment by richardoti** [comment]
:+1: used quite frequently

File: Source/Functions.swift (hunk 1 above)
Comment (richardoti) [inline Source/Functions.swift]: 2long

Comment (zenangst) [inline Source/Functions.swift]: It's just `dispatch` now.

**Pull request merged**